guide-us-english.tex

```diff
@@ -305,10 +283,17 @@ \subsection{Scaling the SoS}
 
 \subsection{The Executive Action Team}
 The Scrum of Scrums for the entire agile organization is called the
-\textbf{Executive Action Team (EAT)}. The EAT is the final stop for
+\textbf{Executive Action Team (EAT)}. This leadership team creates an agile bubble
```

Comment:
"The leadership team creates an agile bubble in the organization where the reference model operates with its own..."